front-end/src/app/reports/report-list/report-list.component.html

```diff
@@ -0,0 +1,79 @@
+<div class="card">
+  <p-toolbar styleClass="mb-4">
+    <ng-template pTemplate="left"> Manage reports </ng-template>
+
+    <ng-template pTemplate="right">
+      <button
+        pButton
+        pRipple
+        label="Create a new report"
+        icon="pi pi-plus"
+        class="p-button-success mr-2"
+        (click)="addItem()"
+      ></button>
+    </ng-template>
+  </p-toolbar>
+
+  <p-table
+    #dt
+    [value]="items"
+    [lazy]="true"
+    (onLazyLoad)="loadTableItems($event)"
+    responsiveLayout="scroll"
+    dataKey="id"
+    [paginator]="true"
+    [rows]="10"
+    [totalRecords]="totalItems"
+    [loading]="loading"
+    currentPageReportTemplate="Showing {first} to {last} of {totalRecords} reports"
+    [showCurrentPageReport]="true"
+  >
+    <ng-template pTemplate="caption">
+      <div class="flex align-items-center justify-content-between">
+        <h5 class="m-0">Recent reports</h5>
+      </div>
+    </ng-template>
+    <ng-template pTemplate="header">
+      <tr role="row">
+        <th pSortableColumn="TBD" id="form-type-column" role="columnheader">
```

Comment:
can this be unset instead of an invalid value?

Reply:
or does that change the functionality?

Reply:
The primeng widget needs to have some sort of string identifier declared (not just an empty string) or else the sort button becomes disabled.